llvm/test/CodeGen/NVPTX/fexp2.ll

```diff
@@ -0,0 +1,61 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -march=nvptx64 -mcpu=sm_20 -mattr=+ptx32 -nvptx-approx-exp2f32 | FileCheck --check-prefixes=CHECK %s
+; RUN: %if ptxas-12.0 %{ llc < %s -march=nvptx64 -mcpu=sm_20 -mattr=+ptx86 -nvptx-approx-exp2f32 | %ptxas-verify -arch=sm_20 %}
```

Comment:
Why is the PTX version different between these two `llc` invocations?

Reply:
Fixed. Thanks.